apps/meeting/server.js

```diff
@@ -60,24 +76,34 @@ const server = require(protocol).createServer(
         const name = query.name;
         const region = query.region || 'us-east-1';
 
+        let client = getClientForMeeting(meetingCache[title]);
         if (!meetingCache[title]) {
-          meetingCache[title] = await chime
-            .createMeeting({
-              ClientRequestToken: uuid(),
-              MediaRegion: region
-            })
-            .promise();
+          let request = {
+            ClientRequestToken: uuidv4(),
+            MediaRegion: region,
+            ExternalMeetingId: title.substring(0, 64),
+          };
+          if (query.ns_es === 'true') {
+            client = chimeRegional;
+            request.MeetingFeatures = {
+              Audio: {
+                // The EchoReduction parameter helps the user enable and use Amazon Echo Reduction.
+                EchoReduction: 'AVAILABLE'
+              } 
+            };
+          }
+          meetingCache[title] = await client.createMeeting(request).promise();
           attendeeCache[title] = {};
         }
         const joinInfo = {
           JoinInfo: {
             Title: title,
             Meeting: meetingCache[title].Meeting,
             Attendee: (
-              await chime
+              await client
                 .createAttendee({
                   MeetingId: meetingCache[title].Meeting.MeetingId,
-                  ExternalUserId: uuid()
+                  ExternalUserId: `${uuidv4().substring(0, 8)}#${query.name}`.substring(0, 64),
```

Comment:
[Understanding]: What is the purpose of this change?

Reply:
change back

Reply:
i will have to change it back as the error `server caught error: ReferenceError: uuid is not defined`. I believe this is because we use regional endpoint here.